cube_convert.py

```diff
@@ -263,9 +269,21 @@
 
     tr.stats.channel = channel_id
 
-    tr.data = tr.data * BITWEIGHT    # Convert from counts to V
-    tr.data = tr.data + offset       # Remove voltage offset
-    tr.data = tr.data / sensitivity  # Convert from V to Pa
+    # Confidence check before we (possibly) modify the data type
+    integer_dtype = np.int32
+    assert tr.data.dtype == integer_dtype
+
+    # KEY: Modifying the time series of integer counts here!
+    if input_args.earthscope:  # Keep in integer counts!
+        if input_args.breakout_box_factor:  # Still need to account for BoB factor
+            tr.data = (tr.data * input_args.breakout_box_factor).astype(integer_dtype)
```

Comment:
I don't think we want to account for the BoB factor in the data conversion. I believe this is generally taken care of as a separate stage in the response file. Perhaps something to confirm with EarthScope that this is the proper way to do it (or maybe ask Ken).

Reply:
I agree and this is how it was done for e.g. Yasur 2016 (https://service.iris.edu/fdsnws/station/1/query?net=3E&sta=YIF1&level=response&format=xml&includecomments=true&nodata=404) — this is 1 / 4.5 = 0.2222, see excerpt:

```xml
<Stage number="2">
    <StageGain>
        <Value>0.222222</Value>
        <Frequency>1.0</Frequency>
    </StageGain>
</Stage>
```

We can update the code here to not accommodate the BoB and instead include that in the separate StationXML file creation script.